packages/core/src/components/popover/popover.md

```diff
@@ -1,281 +1,73 @@
-@# Popovers
-
-<div class="pt-callout pt-intent-primary pt-icon-info-sign">
-    <h5>[Popover2 available in labs package](#labs.popover2)</h5>
-    [`Popover2`](#labs/popover2), the next iteration of this component, is available in the __labs__ package,
-    and uses a new positioning engine called Popper.js to provide much more reliable smart positioning.
-</div>
-
-Popovers display floating content next to a target element.
+@# Popover
+
+**Changes from original [`Popover`](#core/components/popover):**
+
+- [Popper.js](https://popper.js.org) is a massive improvement over [Tether](http://tether.io/) in almost every way!
+  - all the fancy flipping behavior you could want _enabled by default_
+  - endlessly customizable if it isn't perfect _enough_ for you
+  - look, it puts the arrow exactly where it's supposed to be. _every time._
+- all the classic `Popover` features are still supported, with the same names except...
+  - `isModal` &rarr; `hasBackdrop` to match corresponding prop on `Overlay`
+  - `isDisabled` &rarr; `disabled` for consistency with HTML elements
+- ...and except for the handful of Tether-specific props, which are now Popper.js-specific:
+  - `position: Position` &rarr; [`placement: PopperJS.Placement`](#labs/popover2.placement)
+  - `tetherOptions: ITetherOptions` &rarr; [`modifiers: PopperJS.Modifiers`](#labs/popover2.modifiers)
+- ...and one special addition:
+  - `minimal: boolean` applies minimal styles, which includes removing the arrow and minimizing the transition
 
 @reactExample PopoverExample
 
-@## JavaScript API
```

Comment:
I think we likely want to keep this entire section, and all its subsections, as they document key features of `Popover` that are still relevant.